### PR TITLE
sysguage: Add version 6.7.14

### PR DESCRIPTION
**Pull request by wenmin92**
[SysGauge](https://www.sysgauge.com) is a system and performance monitoring utility.

### CODE REVIEW

File: bucket/sysguage.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "hash": "2c31568fecd82108562268bd7536683f9ebcb9ca003d0f1daa29fca387858352"
+        },
+        "32bit": {
+            "url": "http://www.sysgauge.com/setups/sysgauge_setup_v6.7.14.exe#/dl.7z",
+            "hash": "1a5fb536ceb7e893ad852de4e90e672b5a60748a96e9da964a7ae74abbf90508"
+        }
+    },
+    "installer": {
+        "script": [
+            "Rename-Item \"$dir\\bin\\sysgauge.flx\" \"sysgau.flx\"",
```
Comment (Ash258) [inline bucket/sysguage.json]: What??
Why do you put sysgau.flx into current users context??
Use Move-Item $dir\bin\sysgauge.flx $dir

File: bucket/sysguage.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "& \"$dir\\bin\\dsminst.exe\" \"install\" \"$dir\" \"sysgau\"",
```
Comment (Ash258) [inline bucket/sysguage.json]: User `Invoke-ExternalCommand` with `-LogPath`

Comment (Ash258) [inline bucket/sysguage.json]: What does this do? Will it create registry entries? Files? Install drivers??

File: bucket/sysguage.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninstall*\" -Force -Recurse"
+        ]
+    },
+    "shortcuts": [
+        [
+            "bin/sysgau.exe",
```
Comment (Ash258) [inline bucket/sysguage.json]: Backward slashes

File: bucket/sysguage.json (hunk 1 above)
Comment (wenmin92) [reply bucket/sysguage.json]: There is no moving action. `Rename-Item` just rename `sysgauge.flx` to `sysgau.flx`. This action is what `sysgauge_setup_v6.7.14_x64.exe` install script do.

Comment (Ash258) [reply bucket/sysguage.json]:
```suggestion
            "Rename-Item \"$dir\\bin\\sysgauge.flx\" 'sysgau.flx'",
``

File: bucket/sysguage.json (hunk 2 above)
Comment (wenmin92) [reply bucket/sysguage.json]: The install script does this to config the sysgua.flx. 
I used `sysgauge_setup_v6.7.14_x64.exe` to install the app and found this command in the install log. If lack this action, the program can not launch, it says "unable to load product configuration".

Comment (Ash258) [reply bucket/sysguage.json]: > The install script does this to config the sysgua.flx.

What is "this"?

Comment (wenmin92) [reply bucket/sysguage.json]: this is `$dir\bin\dsminst.exe install $dir sysgau`

Comment (Ash258) [reply bucket/sysguage.json]: And what is result of it? It creates some files? registry? something?

Comment (wenmin92) [reply bucket/sysguage.json]: it modifies the `sysgua.flx` file.

Comment (Ash258) [reply bucket/sysguage.json]: How?

Comment (wenmin92) [reply bucket/sysguage.json]: I don't know what changed, this file is a binary file. I just found the file has changed.

Comment (wenmin92) [reply bucket/sysguage.json]: I copy the install log here:
```
Create folder: C:\Program_Files\SysGauge\bin
Create folder: C:\Program_Files\SysGauge\doc
Output folder: C:\Program_Files\SysGauge
Extract: flexense_license.txt... 100%
Output folder: C:\Program_Files\SysGauge\bin
Extract: sysgau.ico... 100%
Output folder: C:\Program_Files\SysGauge\bin
Extract: libpal.dll... 100%
Extract: libdsm.dll... 100%
Extract: dsminst.exe... 100%
Extract: Microsoft.VC80.CRT.manifest... 100%
Extract: msvcr80.dll... 100%
Output folder: C:\Program_Files\SysGauge\bin
Extract: QtCore4.dll... 100%
Extract: QtGui4.dll... 100%
Output folder: C:\Program_Files\SysGauge\bin\imageformats
Extract: qgif4.dll... 100%
Extract: qjpeg4.dll... 100%
Extract: qmng4.dll... 100%
Extract: qsvg4.dll... 100%
Extract: qtiff4.dll... 100%
Extract: Microsoft.VC80.CRT.manifest... 100%
Extract: msvcr80.dll... 100%
Output folder: C:\Program_Files\SysGauge\bin
Extract: libdgg.dll... 100%
Extract: sysgau.exe... 100%
Extract: sysgauge.exe... 100%
Output folder: C:\Program_Files\SysGauge\sounds
Extract: error.wav... 100%
Extract: warning.wav... 100%
Extract: notification.wav... 100%
Extract: message.wav... 100%
Output folder: C:\Program_Files\SysGauge\bin
Extract: sysgauge.flx
Rename: C:\Program_Files\SysGauge\bin\sysgauge.flx->C:\Program_Files\SysGauge\bin\sysgau.flx
Initializing Product Configuration ...
Execute: "C:\Program_Files\SysGauge\bin\dsminst.exe" "install" "C:\Program_Files\SysGauge" "sysgau"
Product Configuration Successfully Completed.
Create folder: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\SysGauge
Create shortcut: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\SysGauge\SysGauge.lnk
Create shortcut: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\SysGauge\License.lnk
Create shortcut: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\SysGauge\Uninstall.lnk
Created uninstaller: C:\Program_Files\SysGauge\uninstall.exe
Completed
```

File: bucket/sysguage.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        },
+        "32bit": {
+            "url": "http://www.sysgauge.com/setups/sysgauge_setup_v6.7.14.exe#/dl.7z",
+            "hash": "1a5fb536ceb7e893ad852de4e90e672b5a60748a96e9da964a7ae74abbf90508"
+        }
+    },
+    "installer": {
+        "script": [
+            "Rename-Item \"$dir\\bin\\sysgauge.flx\" 'sysgau.flx'",
+            "Invoke-ExternalCommand \"$dir\\bin\\dsminst.exe\" \"install $dir sysgau\" -LogPath $dir\\install.log",
```
Comment (Ash258) [inline bucket/sysguage.json]:
```suggestion
            "Invoke-ExternalCommand \"$dir\\bin\\dsminst.exe\" 'install', \"$dir\", 'sysgau' -LogPath $dir\\dsminst.log",
```

File: bucket/sysguage.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninstall*\" -Force -Recurse"
```
Comment (Ash258) [inline bucket/sysguage.json]:
```suggestion
            "Remove-Item \"$dir\\`$*\", \"$dir\\uninstall*\" -Recurse"
```

File: bucket/sysguage.json [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        },
+        "32bit": {
+            "url": "http://www.sysgauge.com/setups/sysgauge_setup_v6.7.14.exe#/dl.7z",
+            "hash": "1a5fb536ceb7e893ad852de4e90e672b5a60748a96e9da964a7ae74abbf90508"
+        }
+    },
+    "installer": {
+        "script": [
+            "Rename-Item \"$dir\\bin\\sysgauge.flx\" 'sysgau.flx'",
+            "Invoke-ExternalCommand \"$dir\\bin\\dsminst.exe\" 'install', \"$dir\", 'sysgau' -LogPath $dir\\dsminst.log",
```
Comment (Ash258) [inline bucket/sysguage.json]: ....

```suggestion
            "Invoke-ExternalCommand \"$dir\\bin\\dsminst.exe\" 'install', \"$dir\", 'sysgau' -LogPath \"$dir\\dsminst.log\" | Out-Null",
```

**Pull request merged**